portfolio/src/main/webapp/index.html

```diff
@@ -7,14 +7,25 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-      <img src="/images/B3913B16-73F9-41BC-BACE-6312C1DBCBA7.JPG" />
+    <nav>
+      <ul>
+          <li><a href='index.html'><strong>Rix</strong></a></li>
+          <li><a href='pages/AboutMe.html'>About Me</a></li>
+          <li><a href='pages/Projects.html'>Projects</a></li>
+          <li><a href='pages/Connect.html'>Connect</a></li>
+          <li><a href='pages/Pictures.html'>Pictures</a></li>
+          <li><a href='pages/ContactMe.html'>Contact Me</a></li>
+          <li><a href='pages/PlayMe.html'>Interactive</a></li>
+      </ul>
+    </nav>
+    
+    <div style="margin-top:25px">
+      <h1>Welcome<br></h1>
+      <div style="margin-left:40px;margin-right:40px">
+      <p><br>So happy to see you here! Please feel free to explore and learn a little bit more about me and my work.<br><br></p>
```

Comment:
Can you line break this at something closer to 80 columns? I tend to be a fan of the [Google standard format](https://google.github.io/styleguide/htmlcssguide.html) where line-breaking is optional but I strongly support.

Reply:
In particular, the Google standard format recommends putting a tag per line.